cloudbuild/run-presubmit-on-k8s.sh

```diff
@@ -14,7 +14,6 @@ LOGS_SINCE_TIME=$(date --iso-8601=seconds)
 
 kubectl run "${POD_NAME}" \
   --image="${IMAGE}" \
-  --requests='cpu=750m,memory=2Gi,ephemeral-storage=2Gi' \
```

Comment:
Does this make presubmit flaky?

Reply:
This is not being used since last few versions as GKE Autopilot has been ignoring this parameter. This line is printed `--requests has been deprecated, has no effect and will be removed in 1.24` in the last few runs

Reply:
Yeah, I think that we need to migrate it to the new API - we added this flag because w/o presubmits were flaky. I propose to move it the separate PR.

Reply:
Reverted, please file a separate PR for this change.